packages/webdriver/tests/utils.test.js

```diff
@@ -44,11 +46,23 @@ describe('utils', () => {
         expect(isValidParameter([1], '(number|string|object)[]')).toBe(true)
         expect(isValidParameter(['1'], '(number|string|object)[]')).toBe(true)
         expect(isValidParameter([{}], '(number|string|object)[]')).toBe(true)
+        expect(isValidParameter([[]], '(number|string|object)[]')).toBe(false)
         expect(isValidParameter([null], '(number|string|object)[]')).toBe(false)
         expect(isValidParameter([false], '(number|string|object)[]')).toBe(false)
         expect(isValidParameter(['1', false], '(number|string|object)[]')).toBe(false)
     })
 
+    it('getArgumentType', () => {
+        expect(getArgumentType(1)).toBe('number')
+        expect(getArgumentType(1.2)).toBe('number')
+        expect(getArgumentType(null)).toBe('null')
+        expect(getArgumentType('text')).toBe('string')
+        expect(getArgumentType({})).toBe('object')
+        expect(getArgumentType([])).toBe('array')
```

Comment:
Array of what?

Reply:
I agree, this is somewhat contradicting with the `[]` suffixed definitions.